### ISSUE TITLE
Difficult to scroll when there are multiple results grids

### ISSUE THREAD

**Issue by MattIrv** [Microsoft/azuredatastudio#3631]
- Azure Data Studio Version: 1.3.8 (also exists in 1.2.4)
Steps to Reproduce:
1. Run a query with multiple result sets, such as `select 1; select 2; select 3; select 4; select 5;`
2. Place your cursor in one of the results grids (not on its header) and try to scroll

Expected behavior: The result sets scroll so that you can see all results
Actual behavior: The result sets do not scroll when your mouse is inside the result grid, only when it is in the result grid header or when it is in the export buttons or scrollbar to the right of the grid

**Comment by craigjar** [comment]
I agree that when results contain multiple queries the experience is not good.  My preference would be that query results data grids be separated into their own tabs and allow you to pin them to keep them open if you decide to run additional queries.

**Comment by anthonydresser** [comment]
This was working at one point, I'll have to check what changed.

**Comment by CoolDadTx** [comment]
With multiple resultsets you may not even be able to see all the results because of the scrolling issue. I have a particular query that returns 4 result sets back. The first couple are fine but trying to scroll to the last 2 tends to either not work at all or doesn't render the expected rows. It is inconsistent.

**Comment by dahln** [comment]
I just downloaded and installed Azure Data Studio today, and I am having the same issue.

**Comment by timothyhoffmann** [comment]
Ditto, scrolling is odd in v1.18.  If you grab the vertical scroll bar and slide down you can see everything but the pane sort of jumps as it scrolls down.

**Comment by kburtram** [comment]
@anthonydresser I'm also seeing the jumpy scrolling behavior when running with multiple resultssets.  For example, if I use the below query and scroll the grids up-and-down the scrolling is noticeably unsmooth.  Do you know if anything changed here?

```
SELECT TOP 100 *
FROM sys.all_objects

SELECT TOP 100 *
FROM sys.all_objects

SELECT TOP 100 *
FROM sys.all_objects

SELECT TOP 100 *
FROM sys.all_objects
```

**Comment by UgmaDevelopment** [comment]
I came here looking for a setting to turn on to allow me to scroll through multiple result panes without stopping at the bottom of the current one, but am I correct in assuming that this is a bug, not a feature?

Thanks for all your efforts! Let me know if I can lend a hand.

**Comment by kunalpgithub** [comment]
I am having same issue. Is there a way we can view multiple result sets as tab (Some widget that we can add.), instead of scrolling?

**Comment by hervehobbes** [comment]
Hello, the problem still exists in version 1.41.2.

**Comment by sharpjs** [comment]
Yeah, I tried using Azure Data Studio this week and hit this scrolling bug basically immediately.  My work regularly involves stored procedures that generate upwards of 15 result sets.  The scrolling bug makes Azure Data Studio almost unusable in this context.

**Comment by alxalm** [comment]
This is a nightmare. Would someone take pity on us poor souls that have to deal with this for a living? It makes life impossible for those that need to regularly work with multiple result sets.

**Comment by AnotherSadGit** [comment]
I've struck this with version 1.46.1.

I've noticed I have to have the mouse pointer over the rather narrow outermost scroll bar (as opposed to the separate scroll bars for each result set) to be able to scroll the whole set of results sets up or down.  This is in contrast to SSMS which will scroll all the results sets if the mouse pointer is anywhere in the result pane.

Even if I scroll with the mouse pointer over the outermost scroll bar it's almost like trying to scroll with a dodgy mouse wheel - it's not smooth all the time, sometimes it seems to stick.  Sometimes it seems to get stuck on a result set and I can't scroll up or down.  But if I scroll aggressively, with large quick movements of the mouse wheel, it seems to get unstuck and I can scroll in either direction past the result set it previously got stuck on.
